grafast/grafserv/src/servers/h3/v1/index.ts

```diff
@@ -227,6 +228,13 @@ export class H3Grafserv extends GrafservBase {
         : ["post"],
     );
 
+    if (this.resolvedPreset.grafserv?.websockets) {
+      app.use(
+        this.dynamicOptions.graphqlPath,
+        defineWebSocketHandler(this.makeWsHandler()),
+      );
+    }
```

Comment:
I've modified this so it only kicks in when websockets are enabled - is this the only place to make that change?

Reply:
Yes, this is the only place.
`addTo` is used only for h3 standalone server.
using with nuxt, we don't call `addTo` but define handlers directly in filesystem router (`server/api/graphql.ts`).